deeptime/markov/msm/tram/_tram.py

```diff
@@ -640,7 +629,7 @@ def _construct_count_models(self, dtraj_fragments):
 
         Parameters
         ----------
-        dtraj_fragments: list(List(Int))
+        dtraj_fragments: list(list(int32))
```

Comment:
int32 technically doesn't exist in python, do you mean `int`? or individual numpy `int32` numbers?